docs/4. Others/Troubleshooting.md

```diff
@@ -0,0 +1,18 @@
+# Troubleshooting
+
+## Introduction
+During development one sometimes may face something, one did not expect to see. For that case we have prepared several tools, which can help you to make things working.
+
+### Cobra Commands
+Those are commands you can execute instead of `serve` to have an interesting result.
+
+1. `routes` - dumps a list of registered routes and handler names. This may help you to understand, why your route is not triggering actions, you expect it to trigger.
```

Comment:
```suggestion
1. `routes` - dumps a list of registered routes and handler names. This may help you understand, why your route is not triggering the actions you expect it to trigger.
```